backend/controller/internal/sql/schema/001_init.sql

```diff
@@ -84,7 +84,7 @@ CREATE TABLE deployment_logs
     verb          VARCHAR,
     time_stamp    TIMESTAMPTZ NOT NULL,
     level         INT         NOT NULL, -- https://opentelemetry.io/docs/specs/otel/logs/data-model/#displaying-severity
-    scope         VARCHAR     NOT NULL,
+    attributes    JSONB       NOT NULL,
```

Comment:
@alecthomas I'm assuming here that we'll want to store these attributes as a blob in the DB, but lemme know if you had something else in mind.

Reply:
Yeah I think this is fine for now. We can optimise later if necessary. We might need runner and deployment ID in this table though, unless they're already there?